packages/crawler/src/crawl-websites.ts

```diff
@@ -0,0 +1,95 @@
+import { crawlerConfig as config } from '../constants';
+import * as puppeteer from 'puppeteer';
+import { evaluatePage } from './evaluate-page';
+import {
+  getAggregatedConsumedTagNames,
+  getAggregatedConsumedTagNamesForVersionsAndPrefixes,
+  getConsumedPrefixesForVersions,
+  getPdsTagNamesWithPropertyNames,
+  getRawDataWithoutVersionsAndPrefixes,
+} from './helpers/convert-data-helper';
+import { writeGeneralReport, writeWebsiteReport } from './helpers/fs-helper';
+import { ConsumedTagNamesForVersionsAndPrefixes, TagNameData, TagNamesWithPropertyNames } from './types';
+import { Page } from 'puppeteer';
+
+export const crawlPage = async (page: Page, websiteUrl: string): Promise<ConsumedTagNamesForVersionsAndPrefixes> => {
```

Comment:
I guess this was extracted to be used in the e2e test which makes sense.
However, naming is a bit misleading with `crawlPage`, `crawlWebsite` and `crawlWebsites`.

Reply:
removed "crawlPage" and added calling "evaluatePage" directly from e2e tests